src/ui/public/chrome/directives/app_switcher/app_switcher.js

```diff
@@ -1,10 +1,18 @@
 import DomLocationProvider from 'ui/dom_location';
 import { parse } from 'url';
-import { bindKey } from 'lodash';
+import { bindKey, filter } from 'lodash';
```

Comment:
`filter` is no longer needed her. We should really update our eslinter to catch unused vars like this (https://github.com/elastic/kibana/issues/7609).